tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper.cs

```diff
@@ -218,7 +219,7 @@ public static void ClearProfilerEnvironmentVariables()
                 environmentVariables["DD_TRACE_DEBUG"] = "1";
             }
 
-            if (!string.IsNullOrEmpty(processToProfile))
+            if (!string.IsNullOrEmpty(processToProfile) && !ignoreProfilerProcessesVar)
```

Comment:
I didn't see you setting `ignoreProfilerProcessesVar = true` anywhere. Are you using this or did I miss something?

Reply:
Can you help me understand why we are skipping this setting for the trimmed app scenario?

Reply:
This is a workaround for a small bug we have in the clr profiler for linux. Currently the linux clr profiler gets the `CurrentProcessName` by looking at `/proc/self/comm` and the value there gets truncated to 15 chars only. `Samples.Trimming` contains 16 chars, so it gets truncated. Because `Samples.Trimming != Samples.Trimmin` then the sample doesn't get instrumented. 🤦🏻